packages/config/src/chains/avalanche.ts

```diff
@@ -6,10 +6,10 @@ export const avalanche: ChainConfig = {
   name: 'avalanche',
   chainId: 43114,
   explorerUrl: 'https://snowtrace.io',
-  explorerApi: {
-    url: 'https://api.snowtrace.io/api',
-    type: 'routescan',
-  },
+  // explorerApi: {
```

Comment:
Why comment this out?

Reply:
This could simply be removed. Right now we're not calling this anywhere